cas-server-webapp/src/main/webapp/WEB-INF/casComponentsMapping.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:util="http://www.springframework.org/schema/util"
```

Comment:
Let me clarify: I understand why you chose a separate file. My concern is: 

1. Should this be something an overlay could do, rather than CAS? 
2. Yet another XML file?
3. Do we ever see the need for a deployer to overlay this new XML file? If so, why not keep it in the deployer, and limit the configuration to one xml file only?